server/src/main/java/org/opensearch/indices/IndicesService.java

```diff
@@ -230,6 +231,19 @@ public class IndicesService extends AbstractLifecycleComponent
         Setting.Property.NodeScope
     );
 
+    /**
+     * Used to specify SEGMENT replication type as the default replication strategy for all indices in a cluster. By default, this is false.
+     */
+    public static final String CLUSTER_SETTING_REPLICATION_TYPE = "cluster.indices.replication.strategy";
+
+    public static final Setting<ReplicationType> CLUSTER_DEFAULT_REPLICATION_TYPE_SETTING = new Setting<>(
```

Comment:
`CLUSTER_REPLICATION_TYPE_SETTING` seems more appropriate here. Default is implied by the default value that will be provided. Also would need documentation around this to demonstrate the hierarchy of `CLUSTER_REPLICATION_TYPE_SETTING` and `INDEX_REPLICATION_TYPE_SETTING`

Reply:
Sure. I will update that. Yes, I will open an issue in documentation repo to document this changes once this PR is merged.